### ISSUE TITLE
How can I create a new ILocalizationService for a new language

### ISSUE THREAD

**Issue by Charmatzis** [IdentityServer/IdentityServer4#46]
Hi, I want to replace the events, the messages and the scopes strings inside the 
[Resources](https://github.com/IdentityServer/IdentityServer4/tree/dev/src/IdentityServer4/Resources ) classes.
How can I do it?

**Comment by leastprivilege** [comment]
We haven't implemented localization in 4 yet. We probably will use the asp.net core built-in capabilities - but not before rc2.

**Comment by PivotalAnimal** [comment]
I like the new localization features of .NET 462. Hopefully that will be ported to Core. Of course, it depends on .resx which means it is compiled as part of the assembly. Maybe it can be dynamically loaded too somehow? [https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/](https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/)

**Comment by brockallen** [comment]
some notes: I investigated the new i18n APIs in core. IStringLocalizer is mainly meant for the host to allow overriding of localizable strings. IdSvr would have to allow/inject this for its resources to allow the host to override. If the host does not, then there's a flag that shows that there's no override, so then we'd need to code for a default value. For the default value, we could then have our own set of localized resx files to provide default localized messages. 

So in short, a bit of work to support this.

**Comment by brockallen** [comment]
We removed it, as it was not needed in the core. The only main place that needs to localize data from the core would be on the error UI. This could then be done via the new IStringLocalizer in ASP.NET Core.

**Comment by leastprivilege** [comment]
that sounds 
![images](https://cloud.githubusercontent.com/assets/1454075/17943964/f1736fb4-6a3d-11e6-8ecc-5595e2b17efe.jpg)
